.github/CODEOWNERS

```
@@ -1 +1 @@
-* @digorithm @hal3e @MujkicA @segfault-magnet @Salka1988 @Br1ght0ne
+* @hal3e @segfault-magnet @xgreenx @Dentosal @MitchTurner @netrome @AurelienFT @rymnc
```

could we just do 
```suggestion
* @FuelLabs/client 
```
?